darc/darc.go

```diff
@@ -706,6 +706,11 @@ func (s Signer) GetPrivate() (kyber.Scalar, error) {
 	}
 }
 
+func (s Signer) String() (string, error) {
+	prk, err := s.GetPrivate()
+	return "{Pub:" + s.Identity().String() + ", Priv:" + prk.String() + "}", err
```

Comment:
I'm not comfortable with making it quite so easy to print out a private key by mistake. Can you explain what problem this new code solves?

Reply:
I am modifying this code so that putting the private and public keys into the JSON message happens in bcadmin.